src/Microsoft.ML.Console/Console.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.ML.Runtime.Tools.Console
+{
+    public static class Console
+    {
+        public static int Main(string[] args)
+        {
+            string all = string.Join(" ", args);
+            return Maml.MainAll(all);
+        }
+
+        public static unsafe int MainRaw(char* psz)
+        {
+            string args = new string(psz);
+            return Maml.MainAll(args);
+        }
```

Comment:
Is `MainRaw` on `char*` a thing? I've not quite seen this pattern before.

Reply:
Oh, these came from MAML.cs. I see that now.

The sole purpose of this DLL is to enable invocations of `dotnet` to work. That's the only reason it exists. So drop the `char*` 

Also: at the risk of stating the obvious, let's not *copy* functions of assemblies we are referencing, let's call them.

The only thing I'd like to see in this class is `public static int Main(string[] args) => Maml.Main(args);`, and that is it.

---
In reply to: [192462330](https://github.com/dotnet/machinelearning/pull/248#discussion_r192462330) [](ancestors = 192462330,192227331)